era/era.go

```diff
@@ -45,6 +45,11 @@ func getCertificate(host string, config []byte, verifyRemoteReport func([]byte)
 	}
 
 	if verifyRemoteReport != nil {
+		if len(quote) == 0 {
+			fmt.Println("ERROR: Received an empty quote from host. Is it running in OE Simulation mode?")
```

Comment:
Pls separate UI and logic. To this end, define an error type and return it. Check for that error in main and then print.